aiida/orm/utils/managers.py

```diff
@@ -14,6 +14,7 @@
 """
 
 from aiida.common.links import LinkType
+from aiida.common import exceptions as exc
```

Comment:
If you don't mind, please remove the alias. Prefer the long version and we don't have the abbreviation `exc` in the code base

Reply:
Sure, for some reason I was sure we were using this pattern, not sure why...